src/nvim/os/unix_defs.h

```diff
@@ -31,12 +31,6 @@
 #ifndef USR_EXRC_FILE
 # define USR_EXRC_FILE "~/.exrc"
 #endif
-#ifndef USR_VIMRC_FILE
-# define USR_VIMRC_FILE "~/.nvimrc"
```

Comment:
It's not your place to question it really.  These are the standards set forth by the operating systems you target.  I'll quote some of Raymond Chen's ironic maxims about his frustration when dealing with software during the MS-DOS to Windows 95 transition:

> 4. Manuals are a waste of time.
> 9. Find a rule and break it as blatantly as possible.
> 19. Second-guess the specification whenever possible.

Please don't keep repeating the same mistakes.  As for support, different locations is trivial to deal with, please don't use it as an excuse to do the wrong thing.

> 12. It is better to be lucky than good.